.github/workflows/triage-helper.yml

```diff
@@ -0,0 +1,26 @@
+name: Issue Triage Helper
+on:
+  issues:
+    types: [labeled]
+jobs:
+  routed:
+    runs-on: ubuntu-latest
+    if: >-
+      contains(
+        fromJSON('[
+          "Status: In Progress",
+          "Status: Backlog",
+          "Status: Needs More Information"
```

Comment:
Why only these labels? The rule should be "Exactly one `Status: *` label on every issue." No?

Reply:
This is the acceptance criteria from the Asana ticket:

>When a triage Status label (Needs Info, Backlog, In Progress) is applied
>Remove all other Status: * labels.

This does exactly that. Mind that the trigger condition is one of these labels.

Reply:
Oh no this is too familiar from my last job. I had a guy on my team who would include obvious typos in copy text because "That's what the acceptance criteria said" 😂 😭

Reply:
Hahahaha, I wasn't trying to be robotic. Was just reminding you that this was the criteria that _we_ came up a few weeks ago. Happy to change but if we want to keep this focused on triage, I think the current criteria makes more sense.